app/models/manageiq/providers/redhat/infra_manager/vm_or_template_shared/scanning.rb

```diff
@@ -151,7 +151,7 @@ def connect_to_ems(ost)
     $log.info "#{log_header}: Connecting to [#{ems_display_text}] for VM:[#{@vm_cfg_file}]"
 
     begin
-      ost.miqRhevm = ext_management_system.rhevm_inventory(:force_v3 => true)
+      ost.miqRhevm = ManageIQ::Providers::Redhat::Inventory::Collector.new(ext_management_system, nil)
```

Comment:
Not for this, but I think we should make the Collector initializer `initialize(manager, target = nil)` in core if it isn't required.